docs/python-api.md

```diff
@@ -1243,6 +1243,18 @@ Also see the {ref}`sec_tables_api_table_collection` summary.
     :special-members: __getitem__
 ```
 
+##### Associated row class
+
+A row returned from an {class}`IndividualTable` is an instance of the following
+simple {mod}`dataclass<python:dataclasses>`. Each attribute in this class matches
```

Comment:
I don't think we want to document them as dataclasses - we might want to change this implementation detail in the future.

Reply:
Yeah, that's a good point.